app/src/androidTest/java/ch/epfl/sdp/peakar/TestMenuBarActivityTest.java

```diff
@@ -0,0 +1,62 @@
+package ch.epfl.sdp.peakar;
```

Comment:
The file should be moved to the corresponding package. We should probably also create a utils package for the test helpers.

Reply:
Since this class is very temporary I'll leave it here. However I created a utils package and moved other helpers there